src/Components/TodoContainer.js

```diff
@@ -1,14 +1,12 @@
-// TodoContainer.js
 import React, { useEffect, useState, useCallback } from "react";
 import TodoList from "./TodoList";
 
 function TodoContainer() {
-    const [todoList, setTodoList] = useState([]);
+    const [todoList, setTodoList] = useState([{ id: '', title: '', createdTime: '' }]);
```

Comment:
nice job making these detailed default values, but you can just make it an empty array `[]` as the default. 

You shouldn't get any `PropType` errors because the components that expect these values to be a detailed object like this shouldn't even render/run until your `isLoading` state is set back to `false`